emcmake.py

```diff
@@ -51,6 +66,15 @@ def has_substr(args, substr):
       print('emcmake: no compatible cmake generator found; Please install ninja or mingw32-make, or specify a generator explicitly using -G', file=sys.stderr)
       return 1
```

Comment:
We have a couple of `return 1` here in this function that are supposed to exit early without running cmake at all. 

I guess you do `if args == 1: return args` in `run` but that doesn't seem create.  Perhaps replacing `return 1` with `sys.exit(1)` would work?

Reply:
Ah that's a good idea,  excellent catch. Fixed as of the latest commit (29fa7fe542bb42495c2734424c870b98403f7aa9).